prepare 6.x for docs release (#213)
neo4j/docs-graphql: modules/ROOT/partials/notes.adoc

# tag::note[]
[CAUTION]
====

> You are looking at version 6 of the GraphQL Library.

Reviewer: ```suggestion
This is the documentation of the GraphQL Library version 6.
```

> For the long-term support (LTS) version 5, refer to link:https://neo4j.com/docs/graphql/5/[GraphQL Library version 5 LTS].

Reviewer 2: ```suggestion
For the long-term support (LTS) version 5, refer to link:{neo4j-docs-base-uri}/graphql/5/[GraphQL Library version 5 LTS].
```
